crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -821,6 +824,7 @@ pub mod wasm_test {
 
     #[tokio::test(flavor = "multi_thread")]
     #[cfg(feature = "slow_tests")]
+    #[cfg_attr(target_os = "macos", ignore = "flaky")]
     async fn enzymatic_session_success() {
```

Comment:
This is the last one left, so I'll give this a shot (I forgot we were each claiming a test)